### ISSUE TITLE
no like or save for localstorage

### ISSUE THREAD

**Issue by izzudin26** [nyancodeid/quotes#159]
this is why i've do pull request with name Adding Like Quote and save it to localstorage
- Forgetting Save quote and forget keyword to searching quote
- Saving quote to storage device but not need to share in social media
- Simple saving quote without spamming on device storage,

**Comment by ngekoding** [comment]
Sorry brothers, I just thinking about the liked quote will make another quote with bad impression.

So what the problem?

It is because we store any types of quotes, from any author and so on. And when the quote is from the prophet (as example) or any related to Islam (shahabah, sheikh, etc...), and the quote have a less like, it is looks not good, that will pushing down the Islam.

I know this is a local storage, but maybe for the future this will go saved on a cloud, and any other feature will comes, like sorting by the most liked, etc.

**Comment by izzudin26** [comment]
> > Sorry brothers, I just thinking about the liked quote will make another quote with bad impression.
> > So what the problem?
> > It is because we store any types of quotes, from any author and so on. And when the quote is from the prophet (as example) or any related to Islam (shahabah, sheikh, etc...), and the quote have a less like, it is looks not good, that will pushing down the Islam.
> > I know this is a local storage, but maybe for the future this will go saved on a cloud, and any other feature will comes, like sorting by the most liked, etc.
> 
> As already mention above, this feature not same as an Upvote or Counted Like. The porpose of this feature is save our favorite quotes. That's why i'm named this feature as `Favorite` rather than `Like`, because this lists of favorite quotes is stored on LocalStorage so we didn't store any kind of data into database or server.
> 
> I had the same interpretation as you at first, only because I was carried away by the word `Like`, so I think about it again and ended up just like, it is favorite feature? and yes i think is favorite. So it seems okay for this feature to be implemented here.

yep maybe just wrong give name for this feature, thank you for accept my pull request :)
